github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```
@@ -528,6 +528,10 @@ external_plugins:
     endpoint: http://botreview:9900
     events:
       - pull_request
+  - name: phased
```

This configuration tells `hook` that it should send pull request events for the `kubevirt/project-infra` repository to the `phased` external plugin.

You can do either of these:
* move the configuration up into the `kubevirt` org section [here](https://github.com/kubevirt/project-infra/pull/3103/files#diff-3a088ffbe61dd30f930cd1855ab72a652fcea4a33c4b047eeb6df38f7a79ad51R500)
* create a new kubevirt/kubevirt section and move the config there.

---

```suggestion
  kubevirt/kubevirt
  - name: phased
```

This would enable the event handling explicitly for kubevirt/kubevirt repo only.

---

moved it to the current kubevirt/kubevirt
is it ok ?

https://github.com/kubevirt/project-infra/compare/0883ae5c8a814d25c1fd7c5b6500f6a1111e600e..c8e7d7a5c351e912b20d3de94018fa6bccee66ec